README.md

```diff
@@ -7,12 +7,18 @@
 `xtevent` is a Stata package to estimate linear panel event-study models. It includes three commands: `xtevent` for estimation; `xteventplot` to create event-study plots and; `xteventtest` for post-estimation hypotheses testing. 
 
 
-- Last version: 2.2.0 (15mar2023)
+- Last version: 3.0.0 (jan2024)

```

Comment:
@Constantino-Carreto-Romero we'll need to make one more push to add the day here.

Reply:
updated in https://github.com/JMSLab/xtevent/pull/168/commits/90d6dbc69456faae71e93088b07b8301d992c9f9